PICS.xcodeproj/xcshareddata/xcschemes/PICS.xcscheme

```diff
@@ -83,7 +83,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "--showOnboarding"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
We possibly do not want to have this in the beta deployment, as it will show onboarding every time for everything. In the real app, we can also delete and re-insntall the app to show the onboarding 😄